src/clients/Deluge.ts

```diff
@@ -91,7 +109,7 @@ export default class Deluge implements TorrentClient {
 			});
 		}
 		try {
-			json = await response.json();
+			json = await (response.json() as Promise<DelugeResponse>);
```

Comment:
```suggestion
			json = await response.json();
```

I'm not entirely sure why this is needed, we instantiate `json` as DelugeResponse, so adding an additional cast in setting it, especially as a Promise, doesn't seem necessary?

I might be wrong....@mmgoodnow said something similar to me in my PR for deluge.

https://github.com/cross-seed/cross-seed/pull/499#discussion_r1357766342